### PR TITLE
HTTP Access Log Combined monitoring

### PR DESCRIPTION
**Pull request by VVelox**
For monitoring logs in the common HTTP Access Log Combined format.

DO NOT DELETE THE UNDERLYING TEXT

#### Please note

> Please read this information carefully. You can run `./lnms dev:check` to check your code before submitting.

- [x] Have you followed our [code guidelines?](https://docs.librenms.org/Developing/Code-Guidelines/)
- [x] If my Pull Request does some changes/fixes/enhancements in the WebUI, I have inserted a screenshot of it.
- [x] If my Pull Request makes discovery/polling/yaml changes, I have added/updated [test data](https://docs.librenms.org/Developing/os/Test-Units/).

#### Testers

If you would like to test this pull request then please run: `./scripts/github-apply <pr_id>`, i.e `./scripts/github-apply 5926`
After you are done testing, you can remove the changes with `./scripts/github-remove`.  If there are schema changes, you can ask on discord how to revert.

### CODE REVIEW

File: doc/Extensions/Applications.md [hunk 1]
```diff
@@ -1002,6 +1002,123 @@ You may need to configure `$server` or `$port`.
...
+
+4. Configure it if neeeded. Uses
+   `/usr/local/etc/http_access_log_combined_extend.json`, unless
+   specified via `-c`. See further below for configuration
+   information.
+
+5. If on large setups where it won't complete in a timely manner, run
+   it via cron.
+```
+4/5 * * * * root /etc/snmp/http_access_log_combined -b -q -w
```
Comment (laf) [inline doc/Extensions/Applications.md:1036]: Is this supposed to be 4/5?

File: doc/Extensions/Applications.md [hunk 2]
```diff
@@ -1002,6 +1002,123 @@ You may need to configure `$server` or `$port`.
...
+
+```
+foo:80-access.log
+foo:80-error.log
+foo:443-access.log
+foo:443-error.log
+bar-access.log
+```
+
+Then the generated the auto generated stuff would be a like below.
```
Comment (laf) [inline doc/Extensions/Applications.md:1088]: Just needs `the generated` removing

**Review by laf — CHANGES_REQUESTED** [review]
Just a couple of doc changes, I'd have done them myself but I wasn't sure if the cron one should be `*/5` or not

File: doc/Extensions/Applications.md (hunk 1 above)
Comment (VVelox) [reply doc/Extensions/Applications.md:1036]: Hmm, yeah, this should be */5 for more generic cron. Being able to do 4/5 is a FreeBSD thing.

File: doc/Extensions/Applications.md (hunk 2 above)
Comment (VVelox) [reply doc/Extensions/Applications.md]: Done!

**Comment by VVelox** [comment]
> Just a couple of doc changes, I'd have done them myself but I wasn't sure if the cron one should be `*/5` or not

Yeah, that should be `*/5`. `4/5` won't work on many cron implementations.

File: doc/Extensions/Applications.md [hunk 3]
```diff
@@ -1002,6 +1002,123 @@ You may need to configure `$server` or `$port`.
...
+}
+```
+
+A manual config would be like below. Note that only `foo` has a error
+log that the size will be checked for and reported via the stat
+`error_size`.
+
+```JSON
+{
+    auto: 0,
```
Comment (Jellyfrog) [inline doc/Extensions/Applications.md:1110]: %

Comment (VVelox) [reply doc/Extensions/Applications.md]: @Jellyfrog derp! good catch, thanks! <3

**Review by Jellyfrog — APPROVED** [review]

**Pull request merged**